packages/web/src/common/store/reachability/sagas.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  reachabilityActions,
+  reachabilitySelectors,
+  getContext
+} from '@audius/common'
+import { delay, take, select, race } from 'typed-redux-saga'
+
+const { getIsReachable } = reachabilitySelectors
+const REACHABILITY_TIMEOUT_MS = 8 * 1000
+
+export function* awaitReachability() {
```

Comment:
I know you didn't change it but this function name feels off to me. It doesn't actually await being in a reachable state, it just gets the current reachability

Maybe we could rename it to something like `getReachability`. We could even have an `awaitReachability` that does
```ts
yield take(reachabilityActions.SET_REACHABLE)
```
for us